app/Models/Author.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Author extends Model
+{
+    use HasFactory, SoftDeletes;
+
+    public function authors()
```

Comment:
The relationship seems wrongly named. Please check.

Reply:
Corrected.